src/messages/MessageBuilder.cpp

```diff
@@ -286,6 +286,32 @@ MessageBuilder::MessageBuilder(SevenTvEventApiUpdateEmoteMessageTag,
     this->message().flags.set(MessageFlag::DoNotTriggerNotification);
 }
 
+MessageBuilder::MessageBuilder(SevenTvEventApiUpdateEmoteSetMessageTag,
+                               const QString &actor,
+                               const QString &emoteSetName)
+    : MessageBuilder()
+{
+    auto text =
+        QString("set the current 7TV emote set to \"%1\".").arg(emoteSetName);
```

Comment:
```suggestion
        QString("switched the active 7TV Emote Set to \"%1\".").arg(emoteSetName);
```

Slightly better wording, maybe?